dsl_parser/framework/parser.py

```diff
@@ -439,6 +439,22 @@ def _calculate_element_graph(self):
                         self.element_graph.add_edge(element, dep)
                     continue
 
+                if predicates == [sibling_predicate]:
+                    # These appear to be generally in order, which means
+                    # that we can cut parsing time by a massive amount for
+                    # plugins which have a lot of DSL elements.
```

Comment:
no reason to mention plugins here, this parses blueprints in general and a "plugin" isn't really a thing at this level of abstraction

Reply:
Comments updated.